unit-tests-jest/numbers.test.js

```diff
@@ -0,0 +1,106 @@
+import { evenNumbers, toDollars, divideBy, multiplyBy } from './numbers';
+
+// describe('evenNumbers(numbers)', () => {
+
+//   it('returns only even numbers', () => {
+//     const input = [1, 2, 3, 4, 5, 6];
+//     const output = evenNumbers(input);
+//     expect(output).toEqual([2, 4, 6]);
+//   });
+// });
+
+describe('evenNumbers', () => {
+  it('returns the even numbers', () => {
+    const numbers = [1, 4, 5, -10, 0];
+    const result = evenNumbers(numbers);
+    expect(result).toEqual([4, -10, 0]);
+  });
+});
+
+describe('evenNumbers', () => {
+  it('Do not return falsey values except 0', () => {
```

Comment:
You should put this test inside the `describe` on line 12. The idea is to "describe" how the `evenNumbers` function works with a series of unit tests